packages/fast-components-styles-msft/src/utilities/color/neutral-focus.ts

```diff
@@ -1,15 +1,92 @@
-import { getSwatch, isDarkMode, Palette, palette, PaletteType } from "./palette";
-import { ColorRecipe, colorRecipeFactory, Swatch } from "./common";
-import { DesignSystem } from "../../design-system";
-
-// These literal values are to remove the dependency on neutralForegroundLight and Dark,
-// and are very temporary as focus is being updated to contrast-based as well.
-export const neutralFocus: ColorRecipe<Swatch> = colorRecipeFactory(
-    (designSystem: DesignSystem): Swatch => {
-        const neutralPalette: Palette = palette(PaletteType.neutral)(designSystem);
-
-        return isDarkMode(designSystem)
-            ? getSwatch(0, neutralPalette)
-            : getSwatch(neutralPalette.length - 5, neutralPalette);
-    }
+import {
+    findClosestSwatchIndex,
+    isDarkMode,
+    Palette,
+    PaletteType,
+    swatchByContrast,
+} from "./palette";
+import { Swatch, SwatchResolver } from "./common";
+import { DesignSystem, DesignSystemResolver } from "../../design-system";
+import { accentPalette, backgroundColor, neutralPalette } from "../design-system";
+
+const targetRatio: number = 3.5;
```

Comment:
should these be exported consts from web utilities or colors?

Reply:
I considered that, but we’d probably add them to the design system to make them configurable. I didn’t want to add that to the interface at this point because of the potential to redesign the color recipes to make them easier to work with. If it looks like something that we want configurable it will be easy to update to later.

Reply:
These numbers come from WAI ARIA and are not subject to change through a design system as that would immediately make values in-accessible.